src/components/Write.js

```diff
@@ -29,6 +29,8 @@ class Write extends Component {
         icon: icon
       })
     })
+      .then(res => res.json())
+      .then(letter => this.props.postSeen(letter.id))
```

Comment:
so this looks like we are marking a letter that a user writes as seen so that same user will not see their own letter?

Reply:
Yup that's exactly it :)